src/client/pages/OfferNew/index.tsx

```diff
@@ -61,6 +62,7 @@ export const OfferNew: React.FC = () => {
     '/:locale(se-en|se|no-en|no|dk-en|dk)/new-member/sign',
   )
   const toggleCheckout = createToggleCheckout(history, currentLocale)
+  const isMobile = useMediaQuery({ maxWidth: 640 })
```

Comment:
I'm not saying we should do this now, because I don't think we should, but imo we should probably have a hook or just some function that returns `isMobile` so we use the same measurements/breakpoints everywhere? Instead of passing `maxWidth` to this hook? Or?

Reply:
I could rename this because I'm more interested in the breakpoint in this case, since we change the layout at 640px the checkout (and add more padding so the Intercom doesn't intersect), if it's mobile or not doesn't really matter :)